src/Microsoft.Azure.SignalR.AspNet/Middleware/NegotiateMiddleware.cs

```diff
@@ -160,7 +160,7 @@ private Task ProcessNegotiationRequest(IOwinContext owinContext, HostContext con
             // add OriginalPath and QueryString when the clients protocol is higher than 2.0, earlier ASP.NET SignalR clients does not support redirect URL with query parameters
             if (!string.IsNullOrEmpty(clientProtocol) && Version.TryParse(clientProtocol, out var version) && version >= ClientSupportQueryStringVersion)
             {
-                var clientRequestId = _connectionRequestIdProvider.GetRequestId();
+                var clientRequestId = _connectionRequestIdProvider.GetRequestId("");
```

Comment:
so for aspnet the id always start with `-`?

Reply:
use a prefix "aspnet"